### PR TITLE
📖 Improve building an extension docs

### LINKED ISSUE #17714: Improve docs for "building an AMP extension"
- [ ] Add instructions for creating "OWNERS" file for component.
- [ ] Add a checklist at the top for a quick list of what to make sure is done sure to do.
- [ ] Ensure that putting new components behind "experiment" flag is required.
- [ ] Add references to existing implementation and special requirements for types of contributions. For instance, if implementing a video-player, we should include special documentation and point to a few existing components to use as model.

### PR DESCRIPTION
**Pull request by alabiaga**
Fixes #17714

### CODE REVIEW

**Comment by alabiaga** [comment]
@aghassemi Ali let me know if the additions/edits make sense or if they are inadequate in fixing the issue. Thanks

**Comment by alabiaga** [comment]
adding Cathy, as Ali is out.

**Comment by alabiaga** [comment]
Thanks Cathy!

File: contributing/building-an-amp-extension.md [hunk 1]
```diff
@@ -630,7 +638,7 @@ required if the validator hasn't been updated yet to allow your newly
 created extension, otherwise people using it in production will
 invalidate all their AMP documents.
 
-You can add your extension as an experiment in the
+Add your extension as an experiment in the
```
[the remark this answers is not in the document]
Comment (alabiaga) [reply contributing/building-an-amp-extension.md]: Good point. Will follow up with this change in another PR. Thanks

**Comment by aghassemi** [comment]
Thanks @alabiaga and @cathyxz. sorry it fell through the cracks on my side.

**Pull request merged**